cmd/run.go

```diff
@@ -0,0 +1,98 @@
+package cmd
+
+import (
+	"github.com/Qovery/do-k8s-replace-notready-nodes/utils"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"os"
+)
+
+var runCmd = &cobra.Command{
+	Use:   "do-nodes",
+	Short: "Recycle all not ready digital ocean clusters nodes",
+	Run: func(cmd *cobra.Command, args []string) {
+		_ = setLogLevel()
+		dryRun, _ := cmd.Flags().GetBool("disable-dry-run")
+		log.Info("Starting DO nodes recycler.")
+
+		if !dryRun {
+			log.Info("Running DO nodes recycler in dry mode.")
+		}
+
+		kubernetesConn, _ := cmd.Flags().GetString("kube-conn")
+		if kubernetesConn != "in" && kubernetesConn != "out" {
+			log.Error("Choose a Kubernetes connection method between 'in' or 'out'")
+			return
+		}
+
+		token, isTokenPresent := os.LookupEnv("DIGITAL_OCEAN_TOKEN")
+		if !isTokenPresent || token == "" {
+			log.Error("You need to add your digital ocean token to env DIGITAL_OCEAN_TOKEN")
+			return
+		}
+
+		clusterId, isIdPresent := os.LookupEnv("DIGITAL_OCEAN_CLUSTER_ID")
+		if !isIdPresent || clusterId == "" {
+			log.Error("You need to add the digital ocean cluster id to env DIGITAL_OCEAN_CLUSTER_ID")
+			return
+		}
+
+		runKubeCmd(cmd, kubernetesConn, dryRun)
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(runCmd)
+	runCmd.Flags().BoolP("disable-dry-run", "y", false, "Disable dry run mode")
+	runCmd.Flags().StringP("kube-conn", "k", "in","Kubernetes connection method, choose between : in/out")
+}
+
+func runKubeCmd(cmd *cobra.Command, kubernetesConn string, dryRun bool) {
+	utils.RunWorkerTest()
+	k8sClientSet, dynamicClient := getKubeClient(cmd, kubernetesConn)
+	DOclient := utils.GetDOClient()
+	utils.WatchNodes(k8sClientSet, dynamicClient, DOclient)
+	if dryRun {
+		//recycleStuckNodes(DOclient, stuckNodes)
+		log.Debug("Starting kubernetes nodes watch.")
+		utils.WatchNodes(k8sClientSet, dynamicClient, DOclient)
+	}
+}
+
+func getKubeClient(cmd *cobra.Command, kubernetesConn string) (*kubernetes.Clientset,dynamic.Interface) {
+	var k8sClientSet *kubernetes.Clientset
+	var dynamicClient dynamic.Interface
+	var err error
+
+	switch kubernetesConn {
+	case "in":
```

Comment:
in is useless here as there are only 2 possibilities that you already checked during the arg check

Reply:
it was legacy code, was afraid to delete it